test/e2e/cypress/e2e/sap_system_details.cy.js

```diff
@@ -132,4 +132,21 @@ context('SAP system details', () => {
       });
     });
   });
+
+  describe('Deregistration', () => {
+    const hostToDeregister = {
+      name: 'vmnwdev02',
+      id: 'fb2c6b8a-9915-5969-a6b7-8b5a42de1971',
+    };
+
+    before(() => {
+      cy.visit(`/sap_systems/${selectedSystem.Id}`);
+      cy.url().should('include', `/sap_systems/${selectedSystem.Id}`);
+    });
+
+    it(`should not include ${hostToDeregister.name} in the list of hosts`, () => {
```

Comment:
We could improve this test (or add a new) to check that the SAP instance belonging to this SAP system is not there. @abravosuse reported some potential issue on this?